defaults/main.yml

```diff
@@ -37,9 +37,9 @@ erigon_host_ip: ""
 erigon_log_verbosity: 3
 erigon_insecure_unlock: 0
 erigon_account_password_file: ""
-erigon_cmdline_args: ""
+erigon_internal_cmdline_args: ""
```

Comment:
May be use `_erigon_internal_cmdline_args` so that convention is different to indicate it is internal variable. Users won't feel it as same as other variables

Reply:
Thats a good idea. I'll do this now